pymc/printing.py

```diff
@@ -31,40 +35,62 @@
 ]
 
 
```

Comment:
I'm guessing that this would close #5793 😅

Reply:
Yeah, that's another benefit of using SymbolicRVs